ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -424,6 +449,28 @@ static void guarded_move_done_cb
   publish ("GroundPosition", GroundPosition);
 }
 
+//Since template is outside the scope of the class we cannot use a member variable and 
+//instead have to use a global one like in the guarded_move_done_cb above.
```

Comment:
Yes, this is fine for now.  Technically the variables are static and not global (since they are not exported).